golem-test-framework/src/components/component_service/mod.rs

```diff
@@ -85,20 +86,17 @@ pub enum PluginServiceClient {
 }
 
 #[async_trait]
-pub trait ComponentService {
-    fn client_protocol(&self) -> GolemClientProtocol;
+trait ComponentServiceInternal {
     fn component_client(&self) -> ComponentServiceClient;
     fn plugin_client(&self) -> PluginServiceClient;
+    fn plugin_wasm_files_service(&self) -> Arc<PluginWasmFilesService>;
+}
 
+#[allow(private_bounds)]
```

Comment:
This is only here because I didn't make ComponentServiceInternal.
This is perfectly fine, but means that nobody outside the crate can implement this trait.
If we want to allow outside implementations, I can make it public.